commands/play.js

```diff
@@ -27,21 +27,21 @@ module.exports = {
 	/** @param {import('discord.js').CommandInteraction & {client: import('discord.js').Client & {music: import('lavaclient').Node}, replyHandler: import('../classes/ReplyHandler.js')}} interaction */
 	async execute(interaction) {
 		if (![ChannelType.GuildText, ChannelType.GuildVoice].includes(interaction.channel.type)) {
-			await interaction.replyHandler.localeError('DISCORD_BOT_UNSUPPORTED_CHANNEL');
+			await interaction.replyHandler.locale('DISCORD_BOT_UNSUPPORTED_CHANNEL', {}, 'error');
```

Comment:
I have seen a similar check on voiceStateUpdate, but was using `warning`, is it `warning` or `error` for those kind of checks?